### PR TITLE
Switch Windows example to PowerShell

### PR DESCRIPTION
**Pull request by davidwrede**
### Proposed changes

Changes example in Getting Started tutorial from the old Windows command prompt over to PowerShell.

### Related issues (optional)

#4823

### CODE REVIEW

**Comment by lukehoban** [comment]
Is it clear somewhere that the user should be using PowerShell?

FWIW - I personally still have a slight preference to just provide the raw text of the file, and tell the user to put that in `index.html` in the `site` folder - as noted in https://github.com/pulumi/docs/pull/4678#discussion_r532931955.  The current guidance, especially for Windows, looks more awkward than necessary.

**Comment by davidwrede** [comment]
Combining the file creation and population steps is really just a little optimization to speed up the non-Pulumi part of the tutorial. I can revert it back, but IMHO, it breaks up the flow if I'm copy and pasting code blocks to then jump out of that for a moment, and then jump back in copying and pasting code. 

And yeah, I'll put in that PowerShell is used in the tutorial. FWIW, Microsoft made [PowerShell the default shell](https://support.microsoft.com/en-us/windows/powershell-is-replacing-command-prompt-fdb690cf-876c-d866-2124-21b6fb29a45f) for Windows 10. Also, I deliberately decided to use `Out-File` rather than the short-hand `>` operator to indicate that this is a PowerShell code block.

**Review by lukehoban — APPROVED** [review]

File: content/docs/get-started/aws/begin.md [hunk 1]
```diff
@@ -60,6 +64,10 @@ If you have previously <a href="https://docs.aws.amazon.com/cli/latest/userguide
 
 If you do not have the AWS CLI installed or plan on using Pulumi from within a CI/CD pipeline, <a href="https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys" target="_blank">retrieve your access key ID and secret access key</a> and then set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables on your workstation.
 
+{{% notes "info" %}}
+If you are using [temporary security credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html), you will also have to supply an `AWS_SESSION_TOKEN` value before you can use Pulumi to create resources on your behalf.
+{{% /notes %}}
```
[the remark this answers is not in the document]
Comment (davidwrede) [reply content/docs/get-started/aws/begin.md:69]: With security info specifically, I've seen many users complain over the years that most getting started tutorials (especially cloud tutorials) assume the user has admin access to everything and can easily install, create, and configure whatever they want. Giving folks a heads up on the minimum required perms/settings, etc. to complete the tutorial is probably what we want to keep inline and then push them off to a more detailed page for more info/complicated scenarios.

Given that reasoning, I'll place the session token and profile info back in the main AWS setup page and then link off to it from here, if people need that info.

File: content/docs/get-started/aws/begin.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ aliases: [
 
 Before you get started using Pulumi, let's run through a few quick steps to ensure your environment is set up correctly.
 
+{{% notes "info" %}}
+All Windows examples assume you are running in PowerShell.
+{{% /notes %}}
```
[the remark this answers is not in the document]
Comment (davidwrede) [reply content/docs/get-started/aws/begin.md:27]: Yup, I tweaked one of the shortcodes to put this below the first Windows code block in this tutorial (i.e. the chocolately install step). Commit coming shortly.

**Pull request merged**